swatch-picker/open-swatch-picker.js

```diff
@@ -0,0 +1,202 @@
+import { oklchToHex, oklchToRgb } from "./color-converters.js";
+
+
+const styles = new CSSStyleSheet()
+styles.replaceSync(/* css */`
+    dialog {
+        border-radius: 1rem;
+
+        &::backdrop {
+            background: rgba(0, 0, 0, 0.5);
+            opacity: 0.75;
+        }
+
+        & div[scale] {
```

Comment:
`&` isn't required when the next combinator is descendant, so you can instead just drop it:



```suggestion
        div[scale] {
```

(Same for other rules below. It's only necessary when the next combinator _isn't descendant, so for e.g. it is required in the `&::backdrop` above).

Reply:
are you sure this is cross broswer compliant? I've had to add it to get nesting to work on safari in the recent past.

Reply:
It's in the modern browsers, going back to Safari 17, so I think it will be okay. Safari 17 is available on Sonoma which is the oldest non-EOL version of macOS.

Reply:
Ok. Removed. If we get complaints...